agent/src/main/java/com/teamscale/jacoco/agent/options/AgentOptions.java

```diff
@@ -203,6 +211,24 @@ public String getOriginalOptionsString() {
 		return originalOptionsString;
 	}
 
+	/**
+	 * Remove parts of the API key for security reasons from the options string.
+	 * String is used for logging purposes.
+	 */
+	public String getObfuscatedOptionsString() {
+		// Example for `getOriginalOptionsString()`:
+		// 		config-file=jacocoagent.properties,teamscale-access-token=unlYgehaYYYhbPAegNWV3WgjOzxkmNHn
```

Comment:
The examples of the options usage should go to the documentation (I think it was README for the agent?)

Reply:
I would move it into the documentation of the method. The example does not provide any advantage over the existing documentation of the `teamscale-access-token`.